security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -260,7 +262,7 @@
     static final String DEFAULT_COOKIE_PATH = "/";
     static final boolean DEFAULT_COOKIE_HTTP_ONLY = true;
     static final boolean DEFAULT_COOKIE_SECURE = false;
-    static final String DEFAULT_COOKIE_SAME_SITE = "Strict";
+    static final String DEFAULT_COOKIE_SAME_SITE = "Lax";
```

Comment:
I think the change in the defaults makes sense with Oracle-apps that use IDCS. Nevertheless I would add a textual warning on startup that the cookie policy is now Lax instead of Strict. The developer might then want to turn off the warning or change to `Strict`, if possible. I wouldn't do this as undocumented/unalerted change. Certainly this has to go into the release notes as well

Reply:
You are right

Reply:
Although this problem is not specific to IDCS, it happens with any OIDC provider.

Reply:
For version 2.0 I will document this in changelog. For version 1.x I will add a warning message.

Reply:
> Although this problem is not specific to IDCS, it happens with any OIDC provider.

any OIDC provider that is on a separate domain

Reply:
True. Maybe I should be more clever about it and just compare frontend host with the host of the OIDC provider?

Reply:
Done and logging an info in case this is triggered.